### DISCUSSION TITLE
How to download the desktop app?

### DISCUSSION THREAD

**Discussion by erwinlauener** [bangle-io/bangle-io#229] [Q&A] [answered]
Can't find the 'install' button on app.bangle.io in the top right corner...

**Comment by kepta** [comment, marked as answer]
Hey thanks for opening this ticket. Here are the steps to download:

1. Go to https://app.bangle.io on **Chrome or any other Chromium browser (not Brave or Firefox)**.
2. Click on the button right next to the url as shown below
<img width="897" alt="image" src="https://user-images.githubusercontent.com/6966254/147491487-05edf76e-a7ba-4ebe-96dc-f8c84f7ae617.png">
3. Click on the install button:
<img width="900" alt="image" src="https://user-images.githubusercontent.com/6966254/147491523-102f3134-3e96-40ec-8e5b-c115e7124ea1.png">
4. Your app should be installed in your desktop.
<img width="923" alt="image" src="https://user-images.githubusercontent.com/6966254/147491607-4f5741f1-69ef-401f-b6a3-693633237468.png">

Let me know if the above works.

Note: Brave and Firefox do not work, see https://github.com/bangle-io/bangle-io/issues/225 and https://github.com/bangle-io/bangle-io/issues/226 for more details.

**Reply by erwinlauener** [reply]
Hi, thx for your answer - doesn't work. Neither in Chrome nor in Firefox...

- Chrome: Unsafe...
- Firefox: No button
- ![Chrome-unsafe](https://user-images.githubusercontent.com/16349445/147492836-38d2ab81-cb60-4739-9afe-0dc0b462e3b4.png)
- ![Firefox-no-button](https://user-images.githubusercontent.com/16349445/147492853-0967bebe-9c74-4a70-8ccd-4ae89d972113.png)

**Reply by BearDooks** [reply]
@erwinlauener I would check your chrome install, or your mac. Chrome is not trusting the SSL cert which works fine for me. 

Possible your machine does not trust 'Lets Encrypt' certs, which may explain why you don't see that button.

**Reply by erwinlauener** [reply]
Thx - you're right: It's an issue of my old MacOS... Still waiting for MacPro ARM 😟
